### PR TITLE
Clarify fix all occurrences

### LINKED ISSUE #4106: Spelling 2.2.0 release
Language opportunities
"Fix all occurences" --> occurrences
And all occurrences where? module?

### PR DESCRIPTION
**Pull request by IvenBach**
Closes #4106 
F5 tested & Unit tested

### CODE REVIEW

File: Rubberduck.Resources/Inspections/InspectionsUI.resx [hunk 1]
```diff
@@ -138,8 +138,8 @@
   <data name="QualifiedSelectionInspection" xml:space="preserve">
     <value>{0}: {1} - {2} {3}.{4}, line {5}</value>
   </data>
-  <data name="QuickFix_All" xml:space="preserve">
-    <value>Fix all occurences</value>
+  <data name="QuickFix_AllOpenProjects" xml:space="preserve">
+    <value>Fix all occurences in all open projects</value>
```
Comment (Vogel612) [inline Rubberduck.Resources/Inspections/InspectionsUI.resx]: #4106 states that "occurences" should be "occurrences". Two "r"

File: Rubberduck.Resources/Inspections/InspectionsUI.resx [hunk 2]
```diff
@@ -138,8 +138,8 @@
   <data name="QualifiedSelectionInspection" xml:space="preserve">
     <value>{0}: {1} - {2} {3}.{4}, line {5}</value>
   </data>
-  <data name="QuickFix_All" xml:space="preserve">
-    <value>Fix all occurences</value>
+  <data name="QuickFix_AllOpenProjects" xml:space="preserve">
+    <value>Fix all occurrences in all open projects</value>
```
Comment (retailcoder) [inline Rubberduck.Resources/Inspections/InspectionsUI.resx]: The xaml binding is against `CanExecuteQuickFixInProject`; IMO "Fix all occurrences in project" would be a more appropriate wording. If the command effectively fixes all occurrences *regardless of the project*, then it's the command that's a misnommer, and needs a fix. "in all open projects" is making the code lie...

Comment (retailcoder) [reply Rubberduck.Resources/Inspections/InspectionsUI.resx]: Nevermind, the binding is against `QuickFixInAllProjectsCommand`, but only visible if the default quickfix can be executed at project-level.

Comment (retailcoder) [reply Rubberduck.Resources/Inspections/InspectionsUI.resx]: I don't like the "all open projects" wording though; only parsed (and thus unlocked) projects would be processed... but making the label make that distinction would look weird. "Fix all" was meant to imply "fix all similar results in the above list of inspection results"... I'm not sure #4106 is about clarifying the meaning more than about fixing the spelling of "occurrences".

**Pull request merged**